arkouda_viz/client/arkouda_viz/viz.py

```diff
@@ -0,0 +1,140 @@
+"""Module providing our plotting capabilities."""
+import holoviews as hv
+import panel as pn
+import panel.widgets as pnw
+from arkouda.dataframe import DataFrame
+from typeguard import typechecked
+from holoviews.operation import datashader as ds
+
+pn.extension()
+
+
+def datashade(
+    data: DataFrame,
+    width: int = 500,
+    height: int = 500,
+    engine: str = "bokeh",
+    point_size: int = 1,
+):
+    """Creates an interactive plot that can handle large datasets."""
+    hv.extension(engine)
+
+    data = data.to_pandas().select_dtypes(exclude=["object"])
+
+    def make_data(x_col, y_col):
+        return hv.Points((data[x_col], data[y_col]), label=f"{x_col} vs {y_col}").opts(
+            size=point_size
+        )
+
+    column_names = list(data.columns)
+
+    color_schemes = ["fire", "bgy", "gray", "kbc", "kb", "kgy"]
+
+    x_widget = pn.widgets.Select(
+        name="x data", options=column_names, value=column_names[0]
+    )
+    y_widget = pn.widgets.Select(
+        name="y data", options=column_names, value=column_names[1]
+    )
+    color_widget = pn.widgets.Select(
+        name="color scheme", options=color_schemes, value=color_schemes[0]
+    )
+    enable_slider_checkbox = pn.widgets.Checkbox(name="remove outliers", value=False)
+    z_score_threshold_slider = pn.widgets.FloatSlider(
+        name="z-score threshold", start=0.0, end=5, step=0.1, value=3.0
+    )
+
+    @pn.depends(
+        x_widget.param.value,
+        y_widget.param.value,
+        color_widget.param.value,
+        enable_slider_checkbox.param.value,
+        z_score_threshold_slider.param.value,
+    )
+    def update_data(x_col, y_col, color_scheme, enable_slider, z_score_threshold):
+        data_points = make_data(x_col, y_col)
+
+        if enable_slider:
+            z_scores = (
+                data_points.data["y"] - data_points.data["y"].mean()
+            ) / data_points.data["y"].std()
+            data_points = data_points[abs(z_scores) < z_score_threshold]
+
+        shaded_data = ds(data_points, cmap=color_scheme).opts(
+            hv.opts.RGB(width=width, height=height)
+        )
+
+        shaded_data = shaded_data.opts(xlabel=f"{x_col}", ylabel=f"{y_col}")
+
+        z_score_threshold_slider.disabled = not enable_slider
+
+        return shaded_data
+
+    logo = "/home/quinnmoore/Desktop/logo.png"
```

Comment:
Does this need to be updated to the image's path in the arkouda file path?

Reply:
I agree with jake here, this path likely needs to be updated